tcms/testcases/views.py

```diff
@@ -150,11 +140,25 @@ def post(self, request, *args, **kwargs):
 
         context_data = {
             'test_plan': test_plan,
-            'form': form
+            'form': form,
+            'notify_form': notify_form
         }
 
         return render(request, self.template_name, context_data)
 
+    def _create_test_case(self, form, test_plan):
+        """Create new test case"""
+        test_case = TestCase.create(author=self.request.user, values=form.cleaned_data)
+
+        # Assign the case to the plan
+        if test_plan:
+            test_plan.add_case(test_case)
+
+        # Add components into the case
+        for component in form.cleaned_data['component']:
```

Comment:
The template doesn't allow for components to be added. All m2m related objects in new templates are now added via RPC after the object has been created. 

These 2 lines must be removed but make sure to remove `form.component` if it exists, which may trigger changes down the line of inherited forms.

Reply:
Done. I have removed the chunk of code, and I have deleted `component` from `BaseCaseForm`. It basically wasn't used anywhere, since the RPC API does not add componets through forms, but rather works with the objects directly